src/main/resources/api_configuration.json

```diff
@@ -35,6 +35,60 @@
       "method": "GET",
       "pathPattern": "/orders/validate",
       "proxyPath": "/ebsconet/validate"
+    },
```

Comment:
We need to add 2 more enties, the first is GET /validate for Mosaic type, and another is for POST mosaic/orders